office/npp/Portfile

```diff
@@ -0,0 +1,18 @@
+PortSystem          1.0
+name                npp
+version             8.6.4
+categories          office
+maintainers         gmail.com:matthew.yang204
+description         Notepad++ ported to macOS
+long_description    Notepad++ ported to macOS, with all features working. Supports all Windows shortcuts. Update system works.
+platforms           darwin
+license             ShareAlike
+homepage            https://github.com/matthewyang204/NPP-on-Mac
+master_sites        https://github.com/matthewyang204/NPP-on-Mac/releases/download/${version}/Notepad++.zip
```

Comment:
Is there any particular reason why the source code is not published? I'm just wondering what actually goes into a package that is over 400MB in size.